Source/Scene/Scene.js

```diff
@@ -1851,12 +2012,40 @@ define([
     function updatePrimitives(scene) {
         var frameState = scene._frameState;
 
+        scene._groundPrimitives.update(frameState);
+        scene._primitives.update(frameState);
+
+        var shadowMaps = frameState.shadowMaps;
```

Comment:
Maybe move this new code into its own function so all lines in this function are at a similar level of abstraction.

https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#functions